### ISSUE TITLE
Add Mute, Hold, Transfer buttons to the GUI

### ISSUE THREAD

**Issue by markdegrootnl** [eofster/Telephone#256]
It would be convenient if the buttons for actions during the call would be part of the call windows.

**Comment by jdegger** [comment]
Agreed, at first I was not even aware Telephone could do this because of the inconvenient UI.

Don't get me wrong, I like the cleanness a lot but it wouldn't hurt to add 3 simple buttons :).

**Comment by eofster** [comment]
Yes, I plan to add some buttons to the UI to make features more observable. Will try to strike the right balance between the UI clutter and feature observability.
